accelerator/iri_proxy_apis.h

```diff
@@ -0,0 +1,198 @@
+/*
+ * Copyright (C) 2018-2019 BiiLabs Co., Ltd. and Contributors
+ * All Rights Reserved.
+ * This is free software; you can redistribute it and/or modify it under the
+ * terms of the MIT license. A copy of the license can be found in the file
+ * "LICENSE" at the root of this distribution.
+ */
+
+#ifndef ACCELERATOR_IRI_PROXY_APIS_H_
+#define ACCELERATOR_IRI_PROXY_APIS_H_
+
+#include "accelerator/errors.h"
+#include "cclient/api/core/core_api.h"
+#include "cclient/request/requests.h"
+#include "cclient/response/responses.h"
+#include "utils/logger_helper.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @file iri_proxy_apis.h
+ * @brief Implement IRI Proxy APIs
+ *
+ * tangle-accelerator provides major IRI Proxy APIs wrapper for public usage.
```

Comment:
Remove "for public usage".

Reply:
By the way, did you validate the nested proxy calls?

Reply:
This PR haven't finished that part yet. There would be another PR for HTTP request handlers, and the nested proxy calls would be tested after that PR.